scalafix/build.sbt

```diff
@@ -0,0 +1,54 @@
+lazy val V = _root_.scalafix.sbt.BuildInfo
```

Comment:
can we make this part of scio build instead of different project?

Reply:
Probably not. The build for scalafix rules is pretty specific. I followed their tutorial and creating a separate project seems to be the recommended method